### ISSUE TITLE
Add support for non-scalar values

### ISSUE THREAD

**Issue by patriksvensson** [spectresystems/spectre.cli#3]

**Comment by orjan** [comment]
What kind of format are you looking for, for providing lists?
I've been using CommandLineParser before and they was using `:` separated entries e.g.
`--robots R2-D2:C-3PO`

**Comment by patriksvensson** [comment]
@orjan I'm not sure to be honest. Need to do some research to see what the consensus is on this (if any).

**Comment by orjan** [comment]
Me neither ;)
```
ls postcss.config.js package.json
ansible-playbook playbooks/PLAYBOOK_NAME.yml --limit "host1,host2"
```

Maybe default to space as a separator and make it possible to override in the future if someone needs it?

**Comment by patriksvensson** [comment]
@orjan Yes, I was thinking the same. Only problem then is that an argument could be interpreted as a command and exit prematurely, so there need to be some kind of mechanism to specify "these are arguments, do not interpret them as a command".

Not sure if I make any sense 😆

**Comment by patriksvensson** [comment]
Closed via #64
